<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>dir</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QTreeView" name="treeView"/>
    </item>
    <item row="2" column="1">
     <widget class="QListWidget" name="listWidget"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit">
      <property name="placeholderText">
       <string>set path</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionset_path">
   <property name="text">
    <string>set path</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
